app/proto/conversations.proto

```diff
@@ -84,22 +80,28 @@ message MessageContentChatCreated {}
 message MessageContentChatEdited {}
 
 message MessageContentUserInvited {
-  uint64 target_user_id = 1;
+  int64 target_user_id = 1;
```

Comment:
Can these ever be negative? Why the switch to int?

Reply:
More recent proto files like communities use int64 so I changed it to match

Reply:
Also, I think the reason was js doesn't support uints so int64 is closer?